src/components/Form.js

```diff
@@ -0,0 +1,37 @@
+import { Component, PropType } from 'react'
+
+class Form extends Component {
```

Comment:
You should be careful calling a React Component something like `Form` as there is a built in component named `form` as part of the React DOM. But nice pulling this out. It might be good to specify the prop types though.